### ISSUE TITLE
maybe there is a softmax bug

### ISSUE THREAD

**Issue by dongzhuoyao** [chenxi116/TF-deeplab#7]
the operation already include a softmax function. tf.nn.sparse_softmax_cross_entropy_with_logits.

and before sparse_softmax_cross_entropy_with_logits you also have a softmax, which might slight influence the result.

**Comment by chenxi116** [comment]
In evaluation mode, self.pred is computed from self.logits followed by softmax.
In training mode, the remain_pred used in sparse_softmax_cross_entropy_with_logits is also computed from self.logits. I don't think there are two softmax used in either mode.
